pyproject.toml

```diff
@@ -1,16 +1,17 @@
 [build-system]
 requires = [
     "setuptools",
-    "scikit-build",
+    "scikit-build-core",
     "cython>=3",
     "cmake",
     "ninja",
```

Comment:
These are managed by scikit-build-core and shall not be used here.
If there are any requirements on their version, the scikit-build-core section of pyproject.toml should be used to express those.

Reply:
Done